.travis.yml

```diff
@@ -23,14 +23,18 @@ jobs:
       env: WEBPACK_VERSION=latest JOB_PART=lint
       script: npm run travis:$JOB_PART
     - <<: *test-latest
-      node_js: 8
+      node_js: 10
       env: WEBPACK_VERSION=latest JOB_PART=coverage
       script: npm run travis:$JOB_PART
       after_success: 'bash <(curl -s https://codecov.io/bash)'
+    - <<: *test-latest
+      node_js: 10
+      env: WEBPACK_VERSION=latest JOB_PART=test
+      script: npm run travis:$JOB_PART
     - stage: Webpack canary
       before_script: npm i --no-save git://github.com/webpack/webpack.git#master
       script: npm run travis:$JOB_PART
-      node_js: 8
+      node_js: 10
```

Comment:
I think canary tests should be on minimum supported node, i.e. `6`

Reply:
Done